api/src/pcapi/core/educational/api/offer.py

```diff
@@ -311,6 +313,7 @@ def create_collective_offer(
         interventionArea=offer_data.intervention_area or [],
         templateId=offer_data.template_id,
         formats=offer_data.formats,  # type: ignore[arg-type]
+        author=current_user,
```

Comment:
Ici aussi, le créateur est le paramètre `user` déjà reçu dans la fonction : 
```suggestion
        author=user,
```